### ISSUE TITLE
Add "lazy" option to FILESYSTEM candidate generator

### ISSUE THREAD

**Issue by zkry** [zkry/p-search#38]
During some search situations, it's extremely expensive for p-search to do the initial candidate enumeration. There should be an option for the filesystem candidate generator to not do any initial enumeration on the candidates, but instead, only when the term-count enumeration is performed, the candidate is added.

**Comment by wdkrnls** [comment]
Yes, IMO being lazy or waiting until pressing a "search now" key should be the default behavior.

**Comment by wdkrnls** [comment]
This could be for example `C-c C-c` or `s`.
